From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: libxml2: reader3 example needs LIBXML_OUTPUT_ENABLED

When compiling --without-output, reader3 fails like this:

reader3.o: In function `main':
/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270/build-target/libxml2-2.6.32/doc/examples/reader3.c:95: undefined reference to `xmlDocDump'
collect2: ld returned 1 exit status
make[4]: *** [reader3] Error 1
make[4]: Leaving directory `/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270/build-target/libxml2-2.6.32/doc/examples'

This patch makes the content of the example dependend on
LIBXML_OUTPUT_ENABLED.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 doc/examples/reader3.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libxml2-2.6.32/doc/examples/reader3.c
===================================================================
--- libxml2-2.6.32.orig/doc/examples/reader3.c
+++ libxml2-2.6.32/doc/examples/reader3.c
@@ -15,7 +15,7 @@
 #include <stdio.h>
 #include <libxml/xmlreader.h>
 
-#if defined(LIBXML_READER_ENABLED) && defined(LIBXML_PATTERN_ENABLED)
+#if defined(LIBXML_READER_ENABLED) && defined(LIBXML_PATTERN_ENABLED) && defined(LIBXML_OUTPUT_ENABLED)
 
 /**
  * streamFile:
